gauss_jordan.go

```diff
@@ -108,7 +108,7 @@ func GaussJordan(linearSystem [][]*Int, linearSystemResult []*Int) ([]*Int, erro
 			for i := index_current_row + 1; i < nrows; i++ {
 				factor := new(Int).Mul(extendedMatrix[i][index_current_column], ModInverse(extendedMatrix[index_current_row][index_current_column]))
 				/* Fill with zeros the lower part of pivot column: */
```

Comment:
Please, take this chance to change these comments to the code style `// Fill...`, when/if changing anything else.

Reply:
for sure!